dpnp/random/_random.pyx

```diff
@@ -125,346 +124,427 @@ cpdef dparray dpnp_uniform(long low, long high, size, dtype=numpy.int32):
 
     return result
 
-
-def rand(d0, *dn):
+cdef class RandomState:
```

Comment:
Why you need this class? all functions are used as standalone functions

Reply:
We need this class for our numpy-like interface. see https://numpy.org/doc/stable/reference/random/legacy.html#numpy.random.RandomState. But I think we can redesign it little bit

Reply:
little reproduce
``` python
>>> from dpnp.random import seed, randint, RandomState
Running on: Intel(R) Core(TM) i7-10710U CPU @ 1.10GHz
queue initialization time: 0.108775 (sec.)
SYCL kernels link time: 0.0850323 (sec.)
>>> # first interface
>>> seed(3); print(randint(1,100,9))
24 59 54 13 37 91 60 47 62 <__str__ TODO>
>>> seed(3); print(randint(1,100,9))
24 59 54 13 37 91 60 47 62 <__str__ TODO>
>>> # second interface
>>> rs = RandomState(3)
>>> print(rs.randint(1,100,9))
24 59 54 13 37 91 60 47 62 <__str__ TODO>
````

Reply:
as I found, there are many same functions from diffrent classes.
Example is `rand()`
1. in matlib. https://numpy.org/doc/stable/reference/generated/numpy.matlib.rand.html#numpy.matlib.rand
2. in randomstate https://numpy.org/doc/stable/reference/random/generated/numpy.random.RandomState.rand.html#numpy.random.RandomState.rand
I suspect there many more such cases.

Implement functions at top level (as it numpy.randn()) and later, call them inside this class.
if you need to keep `seed` somewhere - keep it inside C++ backend (not in Cython)

Reply:
The best place for this class is in random module python level interface (with documentation). in Cython we implement real algorithms.

Reply:
I really don't understand why we need python level, when we have cython already? In cupy, numpy there is no problems with docs. Can you, please, clarify this?

Reply:
> Implement functions at top level (as it numpy.randn()) and later, call them inside this class.
> if you need to keep `seed` somewhere - keep it inside C++ backend (not in Cython)
In this case it is needed duplicate docstrings for all functions in the class. I think, current impl is better